osc/output/input.py

```diff
@@ -0,0 +1,51 @@
+import sys
+import textwrap
+from typing import Dict
+from typing import Optional
+
+from .. import oscerr
+from .tty import colorize
+
+
+def get_user_input(question: str, answers: Dict[str, str], default_answer: Optional[str] = None):
```

Comment:
```suggestion
def get_user_input(question: str, answers: Dict[str, str], default_answer: Optional[str] = None) -> Optional[str]:
```

Reply:
It should be `-> str` as explained above.